CppCoreGuidelines.md

```diff
@@ -9932,11 +9936,21 @@ is going to be surprising for many programmers.
 
 ##### Reason
 
-Signed types support modulo arithmetic without surprises from lack of sign bits.
+Even when the values are known to be nonnegative, arithmetic expressions
+involving unsigned integer types may produce unexpected results, see also ES.100
 
 ##### Example
 
-    ???
+    unsigned x = 100;
+    unsigned y = 102;
+    cout << x-y << '\n'; // large value, such as 4294967294
+    cout << abs(x-y) << '\n'; // compile-time error,
+    // old implementations may produce a large floating-point(!) value
+
+    int x = 100;
+    int y = 102;
+    cout << abs(x-y) << '\n'; // prints -2
```

Comment:
Shouldn't that line read `cout << x-y << '\n'; // prints -2` (without the `abs`)?